src/main/resources/select-options-by-index.js

```diff
@@ -14,15 +14,20 @@
     return {disabledOptions: disabledOptionsIndexes};
   }
 
+  const previousSelectedIndex = select.selectedIndex;
```

Comment:
I see... selectedOptions is not supported in IE. Let me see how it can be done

Reply:
@cocorossello Probably not a big deal. All this JS code for selects doesn't work in IE anyway. :) :)

Reply:
This should work, not a lot more code. I used JSON.stringify to compare the arrays, it should be ok, I think. Let me know if you think otherwise.

Also, there is a bit of copy/paste in those files, if you think this is a problem I can think of a workaround